.travis.yml

```diff
@@ -27,24 +39,28 @@ before_install:
 install:
   # Upgrade pip and setuptools. Mock has issues with the default version of
   # setuptools
-  - |
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-    pip install $NUMPY
-    pip install $MPL
-    pip install -r requirements.txt
+  - pip install --upgrade pip
+  - pip install --upgrade setuptools
+
+  - pip install $NUMPY
+  - pip install $MPL
```

Comment:
You can use wheel caching on linux with travis. This builds the wheel the first time a package is used. You can also use the wheel house like we are doing in Matplotlib